safe_cli/operators/safe_tx_service_operator.py

```diff
@@ -406,3 +411,58 @@ def drain(self, to: ChecksumAddress):
             print_formatted_text(
                 HTML("<ansigreen>Safe account is currently empty</ansigreen>")
             )
+
+    def search_account(
+        self, address: ChecksumAddress
+    ) -> Optional[Union[LocalAccount, HwWallet]]:
+        """
+        Search the provided address between loaded owners
+        :param address:
+        :return: LocalAccount or HwWallet of the provided address
+        """
+        eoa_account = [
+            account for account in self.accounts if account.address == address
+        ]
+        if eoa_account:
+            return eoa_account[0]
+
+        hw_account = [
+            account
+            for account in self.hw_wallet_manager.wallets
+            if account.address == address
+        ]
+        if hw_account:
+            return hw_account[0]
```

Comment:
No need to iterate all accounts. Also you need to import `chain` from `itertools` for my suggestion to work.

```suggestion
        for account in chain(self.accounts, self.hw_wallet_manager.wallets):
            if account.address == address:
                return account
```

Reply:
Nice1